### PR TITLE
Add test for a small newton raphson package

### PR DESCRIPTION
**Pull request by Smit-create**
This tests only on an already-defined function. We will modify this test to accept callbacks once we complete working on #1608 

Related to #992

### CODE REVIEW

File: integration_tests/nrp/nr.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+def newton_raphson(x: f64, c: f64, maxiter: i32) -> f64:
+	h: f64 = func(x, c) / func_prime(x, c)
+	err: f64 = 1e-5
+	i: i32 = 0
+	while abs(func(x, c)) > err and i < maxiter:
+		h = func(x, c) / func_prime(x, c)
+		x = x - h
+		i += 1
```
Comment (certik) [inline integration_tests/nrp/nr.py:19]: Convert tabs to spaces.

**Comment by certik** [comment]
Beautiful, that's exactly what we need. Thanks!

File: integration_tests/nrp/nr.py (hunk 1 above)
Comment (Smit-create) [reply integration_tests/nrp/nr.py:19]: Ah, some wierd issue. I have used a tab for indenting and locally looks fine.

Comment (ADV1K) [reply integration_tests/nrp/nr.py]: We should probably set some pre-commit hooks to autoformat the code when commiting. we can use python black as the formatter.

Comment (certik) [reply integration_tests/nrp/nr.py]: We should use LPython as the formatter. :)

**Review by certik — APPROVED** [review]

File: integration_tests/nrp/nr.py (hunk 1 above)
Comment (Thirumalai-Shaktivel) [reply integration_tests/nrp/nr.py]: Yup!

**Pull request merged**